ZFSin/zfs/lib/libzfs/libzfs_sendrecv.c

```diff
@@ -1301,7 +1286,7 @@ send_progress_thread(void *arg)
 		0, NULL, OPEN_EXISTING, 0, NULL);
 	libzfs_handle_t tmp;
 	if (h != INVALID_HANDLE_VALUE) {
-		tmp.libzfs_fd = h;
+		tmp.libzfs_fd = HTOI(h);
```

Comment:
Another `#ifdef _WIN32` case. The progress_thread must re-open the `/dev/zfs` device node to be allowed to issue `ioctl()s` on it. (Or it will block to wait for `ioctl(send);` to finish, rendering 'progress' useless).
This could be changed to `libzfs_open()` if that is considered cleaner.